pkg/kgo/source.go

```diff
@@ -1761,19 +1787,20 @@ func recordToRecord(
 			Value: kv.Value,
 		})
 	}
+	r := recordsPool.get()
```

Comment:
In that case, instead of resorting to the `*sync.Pool`, the record object would be allocate as usual.

https://github.com/grafana/franz-go/pull/1/files#diff-c6763587c65f4655271050afcde2ba38165b48756f75072ab54f3093246efc61R28-R30

Reply:
ah right, thx